lib/src/jpegr.cpp

```diff
@@ -498,6 +499,22 @@ uhdr_error_info_t JpegR::convertYuv(uhdr_raw_image_t* image, uhdr_color_gamut_t
   return status;
 }
 
+float JpegR::getMasteringDisplayMaxLuminance(uhdr_color_transfer_t transfer) {
+  switch (transfer) {
+    case UHDR_CT_LINEAR:
+      return mMaxDispBrightness;
+    case UHDR_CT_HLG:
+      return mMaxDispBrightness != -1.0f ? mMaxDispBrightness : kHlgMaxNits;
+    case UHDR_CT_PQ:
```

Comment:
Does it make sense to make QP max brightness also configurable?

Reply:
Thank you. I have addressed this.